test/instructions.cc

```diff
@@ -14,6 +14,8 @@
 
 #include "test.h"
 
+#include "test.h"
```

Comment:
duplicated include

Reply:
Thanks, fixed this (and a few more tests)